suites/apis/guppyTest.js

```diff
@@ -0,0 +1,87 @@
+Feature('Guppy')
+
+const chai = require('chai');
+const expect = chai.expect;
+const apiUtil = require('../../utils/apiUtil.js');
+const guppy = require('../../services/apis/guppy/guppyService.js');
+const sheepdog = require('../../services/apis/sheepdog/sheepdogService.js');
+const nodes = require('../../utils/nodes.js');
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+Before( async (home, etl, nodes) => {
+  home.complete.login();
```

Comment:
for API test you don't need to do this, you can get access token from a user, see comments below